LayoutTests/fast/scrolling/mac/keyboard-scrolling-with-mouse-scroll.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ useFlexibleViewport=true EventHandlerDrivenSmoothKeyboardScrollingEnabled=true ] -->
+
+<html>
+
+<head>
+    <script src="../../../resources/ui-helper.js"></script>
+    <script src="../../../resources/js-test-pre.js"></script>
```

Comment:
You can use `../../../resources/js-test.js` instead of the `pre`/`post` versions.